frontend/packages/ux-editor/src/components/Elements/Elements.tsx

```diff
@@ -9,17 +9,27 @@ import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmen
 import classes from './Elements.module.css';
 
 import { useCustomReceiptLayoutSetName } from 'app-shared/hooks/useCustomReceiptLayoutSetName';
+import { useProcessTaskTypeQuery } from '../../hooks/queries/useProcessTaskTypeQuery';
 
 export const Elements = () => {
   const { org, app } = useStudioEnvironmentParams();
   const { selectedFormLayoutSetName, selectedFormLayoutName } = useAppContext();
+  const { data: processTaskType } = useProcessTaskTypeQuery(org, app, selectedFormLayoutSetName);
```

Comment:
Should we display a spinner or something similar while this is loading? This query might take some time to complete with a poor internet connection. Should we also add error handling as well? 😊